en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">granularity</td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Granularity value
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getErrorsByCategory</strong></td>
+<td valign="top"><a href="#errorsbycategory">ErrorsByCategory</a></td>
+<td>
+
+Returns the number of proxy errors (total of both 4xx and 5xx response codes) hit count by each error category for
+each time interval (calculates according to the granularity value) within the specified time range for the given
+combination of tenant ID, environment ID, and organization ID. Optionally, you can configure the
+`errorsByCategoryFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">errorsByCategoryFilter</td>
+<td valign="top"><a href="#errorsbycategoryfilter">ErrorsByCategoryFilter</a></td>
+<td>
+
+Errors by category filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getErrorsDetails</strong></td>
+<td valign="top"><a href="#detailsoferrors">DetailsOfErrors</a></td>
+<td>
+
+Returns details of each proxy error (for both 4xx and 5xx response codes) within the specified time range for the
+given combination of tenant ID, environment ID, and organization ID. Optionally, you can configure the
+`errorsDetailsFilter` parameter to further filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">errorsDetailsFilter</td>
+<td valign="top"><a href="#errorsdetailsfilter">ErrorsDetailsFilter</a></td>
+<td>
+
+Error details filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">dataFilter</td>
+<td valign="top"><a href="#datafilter">DataFilter</a>!</td>
+<td>
+
+Data filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getProxyTargetErrorsOverTime</strong></td>
+<td valign="top">[<a href="#errorsbystatuscodecategory">ErrorsByStatusCodeCategory</a>]</td>
+<td>
+
+Returns a list of hit counts for proxy and target errors by each time interval (calculates according to the
+granularity value) within the specified time range for the given combination of tenant ID, environment ID, and
+organization ID. Each proxy and target error category result set contains breakdown of errors as 4xx, 5xx and total.
```

Comment:
Suggestion for last sentence: "The errors is each proxy and target result set are grouped as `4xx`, `5xx`, and `total`."